go/chat/mobilepush.go

```diff
@@ -75,79 +72,11 @@ func (h *MobilePush) AckNotificationSuccess(ctx context.Context, pushIDs []strin
-			types.InboxSourceDataSourceAll, nil,
-			&chat1.GetInboxLocalQuery{
-				ConvIDs: []chat1.ConversationID{convID},
-			}, nil)
-		if err != nil || len(ib.Convs) == 0 {
-			h.Debug(ctx, "FormatPushText: failed to unbox conv: %v", convID)
-		} else {
-			channelName = ib.Convs[0].Info.TopicName
-		}
-		if channelName == "" {
-			// Don't give up here, just display the team name only
-			h.Debug(ctx, "FormatPushText: failed to get topicName")
-			return fmt.Sprintf("%s (%s): %s", msg.Valid().SenderUsername,
-				msg.Valid().ClientHeader.TlfName, msg.Valid().MessageBody.Text().Body), nil
-		}
-		return fmt.Sprintf("%s (%s#%s): %s", msg.Valid().SenderUsername,
-			msg.Valid().ClientHeader.TlfName, channelName,
-			msg.Valid().MessageBody.Text().Body), nil
-	default:
-		return fmt.Sprintf("%s: %s", msg.Valid().SenderUsername, msg.Valid().MessageBody.Text().Body), nil
-	}
-}
-
-func (h *MobilePush) formatReactionPush(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID,
-	membersType chat1.ConversationMembersType, msg chat1.MessageUnboxed) (res string, err error) {
-	reaction, err := utils.GetReaction(msg)
-	if err != nil {
-		return res, err
-	}
-	switch membersType {
-	case chat1.ConversationMembersType_TEAM:
-		// Try to get the channel name
-		ib, _, err := h.G().InboxSource.Read(ctx, uid, types.ConversationLocalizerBlocking,
-			types.InboxSourceDataSourceAll, nil,
-			&chat1.GetInboxLocalQuery{
-				ConvIDs: []chat1.ConversationID{convID},
-			}, nil)
-		if err != nil || len(ib.Convs) == 0 {
-			h.Debug(ctx, "FormatPushText: failed to unbox convo, using team only")
-			return emoji.Sprintf("(%s): %s reacted to your message with %v", msg.Valid().ClientHeader.TlfName,
-				msg.Valid().SenderUsername, reaction), nil
-		}
-		return emoji.Sprintf("(%s#%s): %s reacted to your message with %v", msg.Valid().ClientHeader.TlfName,
-			ib.Convs[0].Info.TopicName, msg.Valid().SenderUsername, reaction), nil
-	default:
-		return emoji.Sprintf("%s reacted to your message with %v", msg.Valid().SenderUsername,
-			reaction), nil
-	}
-}
-
-func (h *MobilePush) FormatPushText(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID,
-	membersType chat1.ConversationMembersType, msg chat1.MessageUnboxed) (res string, err error) {
-	defer h.Trace(ctx, func() error { return err }, "FormatPushText: convID: %v", convID)()
-	if !msg.IsValid() {
-		h.Debug(ctx, "FormatPushText: message is not valid")
-		return res, errors.New("invalid message")
-	}
-	switch msg.GetMessageType() {
-	case chat1.MessageType_TEXT:
-		return h.formatTextPush(ctx, uid, convID, membersType, msg)
-	case chat1.MessageType_REACTION:
-		return h.formatReactionPush(ctx, uid, convID, membersType, msg)
-	default:
-		h.Debug(ctx, "FormatPushText: unknown message type: %v", msg.GetMessageType())
-		return res, errors.New("invalid message type for plaintext")
+func GetUserAvatar(username string, srvInfo *keybase1.HttpSrvInfo) string {
```

Comment:
How come this is still here? I see you have it in avatar land too

Reply:
I forgot to remove this!